src/cell.rs

```diff
@@ -47,6 +47,10 @@ pub struct Ref<'a, T: ?Sized + 'a> {
     value: NonNull<T>,
 }
 
+// SAFETY: `Ref<'_, T> acts as a reference.
+unsafe impl<'a, T: ?Sized + 'a> Sync for Ref<'a, T> where T: Sync {}
```

Comment:
I think this is correct, but can we express the bounds as `where for<'a> &'a T: Sync` and `where for<'a> &'a T: Send` instead so we're matching the impls with those of actual references?

Reply:
Sure!

Reply:
Amended commit

Reply:
Actually I would like to double check if this works when `T` is not `'static`

Reply:
seems fine